containerd-shim-v2/start.go

```diff
@@ -36,6 +36,9 @@ func startContainer(ctx context.Context, s *service, c *container) error {
 			return err
 		}
 		go watchSandbox(s)
+
+		// Start watching for oom events
+		go watchOOMEvents(ctx, s)
```

Comment:
Can this `ctx` be relied on?

Reply:
Can you elaborate @awprice?

/cc @lifupan, @bergwolf.

Reply:
Apologies, I should have been more detailed. Could this context be closed unexpectedly during the shim's operation? or should I rely on the `ctx` in `s *service`?

Reply:
Further investigation/testing and I've found that the context I used was the one for the RPC request, and it does in-fact get closed once the request has completed. I've stopped using it and am now using the context in the `*service`.